### DISCUSSION TITLE
ReEDS-Mexico

### DISCUSSION THREAD

**Discussion by F-Sequera** [NREL/ReEDS-2.0#120] [Q&A: Data]
Hello

It would be possible to have access to the Transmission data used in the report: **ReEDS-Mexico: A Capacity Expansion Model of the 
Mexican Power System**?

Kind regards,

**Comment by vcelloho** [comment]
What detail of transmission data are you looking for? Are you looking for the input data or results from that report?

**Reply by F-Sequera** [reply]
Hello

I'm looking for the input data, specifically the data used for the electrical characteristics of the transmission regions.
